src/information-panel/information-panel.scss

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-information-panel-wrapper {
+  position: relative;
+  overflow: hidden;
+}
+
+@mixin iui-information-panel {
+  position: absolute;
+  top: 0;
+  right: ($iui-3xl * 4) * -1; // Width set negative
+  opacity: 0;
+  display: flex;
+  flex-direction: column;
+  width: $iui-3xl * 4;
+  min-width: $iui-3xl * 2;
+  height: 100%;
+  box-sizing: border-box;
+  box-shadow: -1px 0 14px rgba(0, 0, 0, 0.25);
+  transition: right $iui-speed-fast ease-out, opacity $iui-speed-fast ease;
+  z-index: 2; // Needs to be higher than table's resizers.
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+
+  &.iui-information-panel-visible {
+    right: 0;
+    opacity: 1;
+  }
+
+  > .iui-resizer {
+    height: 100%;
+    width: $iui-m;
+    position: absolute;
+    top: 0;
+    left: -17px; // Why?
```

Comment:
Unsure why this needs to be -17px, I would think -16px would work but it does not.

Reply:
After removing translate, left is -8px (which is half the resizer div size, cause the line is in the center of it...)

Reply:
You are correct, removed the translate and now using -8px.